benchmark/cvar_benchmark.py

```diff
@@ -81,4 +73,7 @@ def get_problem_instance() -> cp.Problem:
             ],
         )
 
-        return problem
+        self.problem = problem
+
+    def time_compile_problem(self):
+        self.problem.get_problem_data(solver=cp.SCS)
```

Comment:
Perhaps something like this to make debugging easy.

```suggestion
        self.problem.get_problem_data(solver=cp.SCS)
        

if __name__ == '__main__':
    cvar = CVaRBenchmark()
    cvar.setup()
    cvar.time_compile_problem()        
```